test/integration/narrow/routes/solar-technologies.test.js

```diff
@@ -0,0 +1,112 @@
+    expect(response.payload).toContain('An electrical grid connection')
+    expect(response.payload).toContain('Solar panels')
+    expect(response.payload).toContain('An inverter')
+    expect(response.payload).toContain('A utility meter')
+    expect(response.payload).toContain('A battery')
+    expect(response.payload).toContain('Limit-loading power diverter to heat or cold store')
+  })
+
+  it('no option selected -> show error message', async () => {
+    const postOptions = {
+      method: 'POST',
+      url: `${global.__URLPREFIX__}/solar/solar-technologies`,
+      headers: { cookie: 'crumb=' + crumbToken },
+      payload: { solarTechnologies: '', crumb: crumbToken }
+    }
+
+    const postResponse = await global.__SERVER__.inject(postOptions)
+    expect(postResponse.statusCode).toBe(200)
+    expect(postResponse.payload).toContain('Select what solar technologies your project needs')
+  })
+
+  it('user selects \'Solar panels\' option -> store user response and redirect to /solar-installation', async () => {
+    varList.solarTechnologies = 'Solar panels'
+    const postOptions = {
+      method: 'POST',
+      url: `${global.__URLPREFIX__}/solar/solar-technologies`,
+      headers: { cookie: 'crumb=' + crumbToken },
+      payload: { solarTechnologies: 'Solar panels', crumb: crumbToken }
+    }
+
+    const postResponse = await global.__SERVER__.inject(postOptions)
+    expect(postResponse.statusCode).toBe(302)
+    expect(postResponse.headers.location).toContain('solar-installation')
+  })
+
+  it('user selects any option AND \'Solar panels\' -> store user response and redirect to /solar-installation', async () => {
+    varList.solarTechnologies = ['Solar panels', 'An electrical grid connection']
+    const postOptions = {
+      method: 'POST',
+      url: `${global.__URLPREFIX__}/solar/solar-technologies`,
+      headers: { cookie: 'crumb=' + crumbToken },
+      payload: { solarTechnologies: ['Solar panels', 'An electrical grid connection'], crumb: crumbToken }
+    }
+
+    const postResponse = await global.__SERVER__.inject(postOptions)
+    expect(postResponse.statusCode).toBe(302)
+    expect(postResponse.headers.location).toContain('solar-installation')
+  })
+
+  it('user selects one option WITHOUT \'Solar panels\' -> store user response and redirect to /project-cost', async () => {
+    varList.solarTechnologies = 'A utility meter'
+    const postOptions = {
+      method: 'POST',
+      url: `${global.__URLPREFIX__}/solar/solar-technologies`,
+      headers: { cookie: 'crumb=' + crumbToken },
+      payload: { solarTechnologies: 'A utility meter', crumb: crumbToken }
+    }
+
+    const postResponse = await global.__SERVER__.inject(postOptions)
+    expect(postResponse.statusCode).toBe(302)
+    expect(postResponse.headers.location).toContain('/productivity/solar/project-cost')
+  })
+
+  it('user selects multiple options WITHOUT \'Solar panels\' -> store user response and redirect to /project-cost', async () => {
```

Comment:
I think you have this test twice. you can remove one of it.

Reply:
First test was for selecting one option, second was for testing multiple options. Doesn't need to be deleted as it does test different functionality :)